### DISCUSSION TITLE
Google not crawling site

### DISCUSSION THREAD

**Discussion by excede1** [requarks/wiki#5262] [General]
We have had "index, follow" on our wiki.js: wiki.powersofattorney.com 
Despite this google has not crawled the site in the past 2 years.
I have tried to add a sitemap.xml to the root folder and then submit to google search console but it wont allow me to.

Can anyone please provide any advice on how to get google to index our site please?

**Comment by leonidasfs** [comment]
Is there any update on this? We would love to have the option to add a manually or automatically a sitemap!

**Reply by Philipinho** [reply]
You can now generate sitemap for Wiki.js automatically with this tool https://github.com/hostwiki/wikijs-sitemap.

**Reply by leonidasfs** [reply]
Thanks a lot I will definitely try it,  but hourly regeneration is a bit overkill! Why not set it to weekly ?

**Reply by Philipinho** [reply]
I can make it configurable. Please an issue on the repo so I can keep track.
